core/dropdowndiv.js

```diff
@@ -290,17 +292,19 @@ Blockly.DropDownDiv.showPositionedByField = function(owner,
  * If we can't maintain the container bounds at the primary point, fall-back to the
  * secondary point and position above.
  * @param {Object} owner The object showing the drop-down
+ * @param {boolean} rtl Right-to-left (true) or left-to-right (false).
```

Comment:
I recognize that this is breaking for any developers using this directly. I checked, and both Makecode and Scratch don't call this directly, since this is new, having the RTL here would match widgetdiv (and would be my preferred api), but if we think it's too late to change this API, then happy to throw this at the end.